tests/trinity/core/p2p-proto/bcc/helpers.py

```diff
@@ -0,0 +1,156 @@
+import asyncio
+
+from cytoolz import (
```

Comment:
In an attempt to maintain good pypy support (though we still don't actually support pypy) we've shifted to importing these from `eth_utils.toolz` which appropriately imports from `cytoolz` if it is installed or `toolz` otherwise.